app/src/main/java/org/fossasia/openevent/api/network/OpenEventAPI.java

```diff
@@ -17,21 +16,24 @@
  */
 public interface OpenEventAPI {
 
-    @GET("/speakers")
+
+    //TODO: Fix Hardcoding here
+    @GET("/event/2/speakers")
     void getSpeakers(Callback<SpeakerResponseList> speakerResponseListCallback);
 
-    @GET("/sponsors")
+    @GET("/event/2/sponsors")
     void getSponsors(Callback<SponsorResponseList> speakerResponseListCallback);
 
-    @GET("/sessions")
+    @GET("/event/2/sessions")
     void getSessions(Callback<SessionResponseList> sessionResponseListCallback);
 
-    @GET("/events")
+    @GET("/event")
```

Comment:
It means event id 2. It can be changed but I am not yet sure how we can get the event id required by user.